apps/next-evans/pages/friends.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import { PrismaClient } from '@prisma/client';
+import invariant from 'tiny-invariant';
+import type { GetStaticProps } from 'next';
+import type { friends as Friend } from '@prisma/client';
+import { LANG } from '@/lib/env';
+import FriendsPageHero from '@/components/pages/friends/FriendsPageHero';
+import { t } from '@/../../libs-ts/locale/src';
```

Comment:
this import looks wrong. the app should depend on the locale package and import it with a non-relative path